core/org.wso2.carbon.server/src/main/java/org/wso2/carbon/server/util/Utils.java

```diff
@@ -119,6 +119,8 @@ public static void printUsages() {
         System.out.println("\t-DworkerNode\t\t\tSet this system property when starting as a worker node.");
         System.out.println("\t-Dprofile=<profileName>\t\tStarts the server as the specified profile. e.g. worker profile.");
         System.out.println("\t-DencryptSecrets=true\t\tEncrypt the secrets in deployment Configuration");
+        System.out.println("\t-DoverrideConfigurationAlways=true\t\t Overwrite the Configurations every Time");
```

Comment:
change property key as per above comment, and update the description not to have 'every time'